simple_history/models.py

```diff
@@ -387,6 +387,7 @@ def get_instance(self):
                 excluded_attnames = [
                     model._meta.get_field(field).attname
                     for field in self._history_excluded_fields
+                    if not isinstance(model._meta.get_field(field), ManyToManyField)
```

Comment:
@jordonwii this works? Looks like this excludes ManyToManyFields from being excluded? The included fields are in `attrs` above

Reply:
Right, that's intentional - the goal here is to prevent an M2M field from ending up in the `.values(...)` call on line 395, because then the `.get()` will fail.

Reply:
Ohh I see. Forgot about this code. That makes sense. Want to just add a comment here briefly explaining that?

Reply:
Done. Let me know if it's unclear - wording "exclude a field from the exclusion list we're trying to re-include" is tricky. :)